spec/policies/person_policy_spec.rb

```diff
@@ -0,0 +1,29 @@
+require 'rails_helper'
+
+RSpec.describe PersonPolicy do
+  describe '#admin?' do
+    context 'the person is not signed in' do
+      subject { described_class.new(nil_person, 'any object') }
```

Comment:
Since we don't every test for `nil_person` we can just use `nil` and get rid of the `let` altogether.